ts/parts/Series.ts

```diff
@@ -4177,6 +4179,32 @@ H.Series = H.seriesType<Highcharts.LineSeries>(
             fireEvent(this, 'afterGetExtremes');
         },
 
+        /**
+         * Find and return the first non null point in the data
+         *
+         * @private
+         * @function Highcharts.Series.getFirstNotNullPoint
+         * @param {Array<Highcharts.PointOptionsType>} data
+         *        Array of options for points
+         *
+         * @return {Highcharts.PointOptionsType}
+         */
+        getFirstNotNullPoint: function (
```

Comment:
Nice refactoring, but it would be easier on the brain if the function is called `getFirstValidPoint`.

Reply:
I agree, this was the first name I used :) I changed this to be more accurate - we have `point.isValid()` that really validates a point, here we just check if the point is null. If you think it's not a problem, I will change to `getFirstValidPoint`.

Reply:
Yes, `isValid` and `isNull` are basicly antonyms.